src/LodeRunner.API/src/Core/Startup.cs

```diff
@@ -114,22 +114,30 @@ public static void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 app.UseCors();
             }
-
-            app.Use(async (context, next) =>
-            {
-                context.Response.Headers.Add("X-B3-TraceId", Activity.Current.Context.TraceId.ToString());
-                context.Response.Headers.Add("X-B3-SpanId", Activity.Current.Context.SpanId.ToString());
-
-                // call next middleware handler
-                await next().ConfigureAwait(false);
-            });
+
+            app.Use(async (context, next) =>
```

Comment:
This diff looks identical.
Wondering if there are any whitespace changes here.

Reply:
EOL changes